backend/test/integration/GameStartEvents.integration.test.js

```diff
@@ -28,13 +28,16 @@ describe('GameStartEvents integration tests', () => {
         SocketIOServer.server.close();
     });
 
-    test('integration test start-game 6 players', async (done) => {
-        /* eslint-disable no-await-in-loop */
-        for (let i = 1; i < 6; i += 1) {
-            await connectAndJoin(clientSockets, i, port, lobbyCode);
-        }
-        /* eslint-enable no-await-in-loop */
-        await startGame(clientSockets);
-        done();
-    });
+    // Run integration tests for all possible lobby sizes.
+    for (let playerCount = 6; playerCount <= 7; playerCount += 1) {
```

Comment:
This is great, especially making a separate test() for each playerCount. We just need to remember to update these values when the maximum player count changes